packages/python-pynvim/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://github.com/neovim/pynvim
 TERMUX_PKG_DESCRIPTION="Python client for Neovim"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.5.0
+TERMUX_PKG_VERSION=0.5.2
 TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
This is something I didn't catch in the original PR for this package.
We don't usually use `$TERMUX_PKG_HOMEPAGE` as a substitution in case it changes to a link other than the repository in the future.

It would be appreciated if you could change this out while we're here.
```suggestion
TERMUX_PKG_SRCURL=https://github.com/neovim/pynvim/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
```